### DISCUSSION TITLE
saas文件提交后显示permisson denied 解决方案

### DISCUSSION THREAD

**Discussion by Senptel** [physics-data/faq#98] [Show and tell] [answered]
saas.sh文件在本地运行正确，但是在github上显示`permisson denied`。并且在本地上file saas.sh 显示
`saas.sh:Bourne-Again shell script, ASCII text executable`
原因是/mnt/c/路径下的文件权限和~/下的权限不一样
解决办法：
`cd`到~$目录下
然后重新`git clone`文件，
此时`ls`显示 **docs**  README.md  saas.sh  **scripts**说明saas.sh没有执行权限，此时用`chmod +x saas.sh`给他加上执行权限再重新上传即可

**Comment by heroxbd** [comment, marked as answer]
这个问题在 https://github.com/physics-data/faq/discussions/92#discussioncomment-1263505 也有讨论。
